lib/chat_web/live/main_live/layout/uploader.ex

```diff
@@ -175,10 +175,16 @@ defmodule ChatWeb.MainLive.Layout.Uploader do
           type={@type}
         />
       <% end %>
-      <div class="hidden">
+      <div id={"uploadErrors-" <> Utils.random_id()} class="hidden" phx-update="append">
```

Comment:
it is better to write helper function than show implementation details in markup

Reply:
done